include/Metabox/Base.php

```diff
@@ -301,4 +301,41 @@ protected function render_result( $posts_moved ) {
 			'updated'
 		);
 	}
+
+	protected function render_roles_dropdown( $name = '', $hide_no_role_option = false ) {
+		global $wp_roles;
+		$roles = $wp_roles->roles;
+
+		if ( ! ( $users = wp_cache_get( 'bm_users', 'bulk-wp' ) ) ) {
+			$users = get_users();
+			wp_cache_add( 'bm_users', 'bulk-wp' );
+		}
+
+		$bulk_move = bulk_move();
+
+		$users_by_roles = $bulk_move->helper->get_users_count_by_roles( $users, $roles );
+
+		$field_name  = $this->meta_box_slug;
+		$field_name .= ! empty( $name ) ? '-' . $name : $name;
+		$field_name .= '-roles-list';
+
+		?>
+		<select id="<?php echo $field_name; ?>"
+				name="<?php echo $field_name; ?>">
```

Comment:
Do output sanitization on variables before printing them inside markup.

Reply:
> Do output sanitization on variables before printing them inside markup.

Since these values are not from DB, I'm using `sanitize_html_class()`. The reason for using `sanitize_html_class()`is that the values are used in HTML element attributes.